diff --git a/dist/chart.js b/dist/chart.js
index 22e47a917b6743867ad1c32c7c41f62524166158..e57ce855c54c5fa2acf28ed04c6a19d11e56f591 100644
--- a/dist/chart.js
+++ b/dist/chart.js
@@ -445,7 +445,10 @@ export class Chart extends EventEmitter.EventEmitter {
             return d.data.map(datumDefaults);
         }, (d) => {
             // The key is the function set or other value that uniquely identifies the datum.
-            return d.fn || d.r || d.x || d.text;
+            return JSON.stringify({
+              function: d.fn || d.r || d.x || d.text,
+              graphType: d.graphType || 'interval'
+            });
         });
         // exit
         graphs.exit().remove();
diff --git a/dist/graph-types/polyline.js b/dist/graph-types/polyline.js
index 19ee726008e56aa4efaba6ab8991f7d164148233..e709b18aa2f9fc75b5cb732170574a18fdc6af76 100644
--- a/dist/graph-types/polyline.js
+++ b/dist/graph-types/polyline.js
@@ -12,6 +12,10 @@ export default function polyline(chart) {
             const yRange = chart.meta.yScale.range();
             let yMax = yRange[0];
             let yMin = yRange[1];
+
+            // Fix
+            if(yMax < yMin) [yMax, yMin] = [yMin, yMax];
+
             // workaround, clamp assuming that the bounds are finite but huge
             const diff = yMax - yMin;
             yMax += diff * 1e6;
diff --git a/dist/samplers/interval_worker_pool.js b/dist/samplers/interval_worker_pool.js
index 144edd05b81f40b1005f89c1d10ede198e3f05e2..784870a7bddb9acae548705df397a818c1dcd32e 100644
--- a/dist/samplers/interval_worker_pool.js
+++ b/dist/samplers/interval_worker_pool.js
@@ -1,4 +1,4 @@
-import Worker from 'web-worker';
+// import Worker from 'web-worker';
 export var BackpressureStrategy;
 (function (BackpressureStrategy) {
     BackpressureStrategy["None"] = "none";
